rhombus/scribblings/ref-values.scrbl

```diff
@@ -57,13 +57,24 @@
 }
 
 @doc(
-  reducer.macro 'values($id = $expr, ...)'
+  reducer.macro 'values($id $maybe_annot $expr_or_block, ...)'
+
+  grammar maybe_annot:
+    :: $annot
+    :~ $annot
+    #,(epsilon)
+
+  grammar expr_or_block:
+    $expr
```

Comment:
Should have an `=` here? I would call this non-terminal `init`, or something like that.